Lesson 5 Assignment

sonalaggarwal555 opened the pull request.

sonalaggarwal555 committed:
Lesson 5 Assignment

cherimarie commented:
@sonalaggarwal555 I can't review a PR with 1,692 files changed. Can we please clean this up so that the only file changes are those directly related to the assignment?

sonalaggarwal555 committed:
Assignment 05

sonalaggarwal555 commented:
@cherimarie I have made changes in the assignment.I have deleted all fontawesome files and added them to node modules. I mistakenly merged the PR . But let me know if you want me to revert this PR and reopen the same.

sonalaggarwal555 commented:
Hi Cheri,
I have updated the PR.Here is the link:
https://github.com/sonalaggarwal555/html300/pull/4/files .Please review the
changes.

On Tue, Apr 14, 2020 at 9:04 AM cheri allen <notifications@github.com>
wrote:

> @sonalaggarwal555 <https://github.com/sonalaggarwal555> I can't review a
> PR with 1,692 files changed. Can we please clean this up so that the only
> file changes are those directly related to the assignment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sonalaggarwal555/html300/pull/4#issuecomment-613531420>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALEZHRTQZSYNIKWD4Z7AFJTRMSCQ3ANCNFSM4MDVZY2Q>
> .
>

sonalaggarwal555 commented:
Hello Cheri,

Due to some mistake,i had to delete 'assignment 06' branch and create it
again.I have updated my assignment in that branch and have generated a new
pull request.Please find the below PR link and ignore the one i have
submitted in the assignment.
https://github.com/sonalaggarwal555/html300/pull/6

Thanks,
Sonal

On Sun, Apr 19, 2020 at 10:24 PM Sonal Aggarwal <sonalaggarwal555@gmail.com>
wrote:

> Hi Cheri,
> I have updated the PR.Here is the link:
> https://github.com/sonalaggarwal555/html300/pull/4/files .Please review
> the changes.
>
> On Tue, Apr 14, 2020 at 9:04 AM cheri allen <notifications@github.com>
> wrote:
>
>> @sonalaggarwal555 <https://github.com/sonalaggarwal555> I can't review a
>> PR with 1,692 files changed. Can we please clean this up so that the only
>> file changes are those directly related to the assignment?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/sonalaggarwal555/html300/pull/4#issuecomment-613531420>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALEZHRTQZSYNIKWD4Z7AFJTRMSCQ3ANCNFSM4MDVZY2Q>
>> .
>>
>

cherimarie commented on lesson05/assignment/css/style.scss:
```diff
@@ -3,4 +3,94 @@
+  width:100%;
+  h1{
+  color: white;
+  font-family: HelveticaNeue, "Helvetica Neue";
```
Font family list should be in descending order of specificity, so this should look something like: 
` font-family: "Helvetica Neue",  Helvetica, Arial, sans-serif;`
Give lots of fallback options.

cherimarie commented on lesson05/assignment/index.html:
```diff
@@ -1,15 +1,117 @@
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/brands.min.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/solid.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/fontawesome.min.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/solid.min.css">
```
👍
